### ISSUE TITLE
Run test fail on GCC: "typedef long double _Float64x"

### ISSUE THREAD

**Issue by ladisgin** [UnitTestBot/UTBotCpp#85]
Change main compiler to GCC.
Run generated test for [calc.c](https://github.com/UnitTestBot/UTBotCpp/blob/main/integration-tests/c-example/src/calc.c). 
Problem with float typedefs in stubs

```
/home/utbot/UTBotCpp/integration-tests/c-example/tests/stubs/lib/pointers/function_pointers_stub.h:112:21: error: two or more data types in declaration specifiers
  112 | typedef long double _Float64x;
      |                     ^~~~~~~~~
/home/utbot/UTBotCpp/integration-tests/c-example/tests/stubs/lib/pointers/function_pointers_stub.h:112:1: warning: useless type name in empty declaration
  112 | typedef long double _Float64x;
      | ^~~~~~~
```

**Comment by antipeon** [comment]
https://github.com/UnitTestBot/UTBotCpp/pull/158#discussion_r894698053 - check strange artifact, maybe it causes the problem

**Comment by ladisgin** [comment]
Fixed in #656
